### ISSUE TITLE
Kick doesn't work

### ISSUE THREAD

**Issue by erdnaxeli** [Nerdmaster/ruby-irc-yail#11]
The kick doesn't work. I use the IRCBot class.

Here is my code :

``` ruby
def _in_msg(event)
  return if event.pm?
  return if event.nick != "marvin"

  if event.message =~ /^.+: chamber . of 6 => .*\*BANG\*.*$/
    nick = event.message.scan(/^[^:]+/)[1]
    @irc.kick(nick, event.channel)
  end
end
```

And the error :

> /home/erdnaxeli/.rvm/gems/ruby-1.9.3-p374/gems/net-yail-1.6.0/lib/net/yail/legacy_events.rb:192:in `legacy_process_event': undefined method`reason' for #<Net::YAIL::OutgoingEvent [kick]:0x00000003039528> (NoMethodError)
>        from /home/erdnaxeli/.rvm/gems/ruby-1.9.3-p374/gems/net-yail-1.6.0/lib/net/yail/dispatch.rb:19:in `dispatch'
>        from (eval):3:in`kick'
>        from /home/erdnaxeli/codaz/Sheraf/lib/sheraf.rb:47:in `_in_msg'

**Comment by Nerdmaster** [comment]
I will probably fix this by just scrubbing legacy event support.  This isn't the first time legacy events crashed due to the new event system.

I'll try to get to drop legacy support in the next week or so, and maybe I can pop in a quick fix today or tomorrow.

**Comment by Nerdmaster** [comment]
Fixed and bumped version to 1.6.1
